docs/contributing.md

```diff
@@ -184,6 +189,65 @@ occasionally check the terminal from which you ran `mkdocs serve` for
 any build errors or warnings. Warnings about
 `"GET /versions.json HTTP/1.1" code 404` are expected and harmless.
 
+### How to build the entire website (not usually necessary)
+
+The full `jj` website includes the documentation for several `jj` versions
+(`prerelease`, latest release, and the older releases). The top-level
+URL <https://martinvonz.github.io/jj> redirects to
+<https://martinvonz.github.io/jj/latest>, which in turn redirects to
+the docs for the last stable version.
+
+The different versions of documentation are managed and deployed with
+[`mike`](https://github.com/jimporter/mike), which can be run with
+`poetry run -- mike`.
+
+On a POSIX system, one way to build the entire website is as follows
```

Comment:
Minor suggestion: On a POSIX system or WSL, one way to build the entire website is as follows.

Reply:
Thanks!